app/workers/updates_synchronizer_worker.rb

```diff
@@ -1,20 +1,50 @@
 class UpdatesSynchronizerWorker
   include Sidekiq::Worker
 
+  TRY_AGAIN_IN = 20.minutes
+  CUT_OFF_TIME = '07:30'.freeze
```

Comment:
Should we put this in the environment? Might make it a bit easier to configure? Maybe calling it CDS_UPDATES_RETRY_UNTIL and CDS_UPDATES_RETRY_EVERY?

Reply:
I'm a bit wary of adding too much configuration - if its something we end up needing we can add it in env vars easily enough later on but considering the other timings are held in the codebase (`sidekiq.yml`) - I think these should also be in the codebase for now.